python/geotag_from_gpx.py

```diff
@@ -78,8 +80,11 @@ def add_exif_using_timestamp(filename, time, points, offset_time=0):
         exiv_lat = (make_fraction(lat_deg[0],1), make_fraction(int(lat_deg[1]),1), make_fraction(int(lat_deg[2]*1000000),1000000))
         exiv_lon = (make_fraction(lon_deg[0],1), make_fraction(int(lon_deg[1]),1), make_fraction(int(lon_deg[2]*1000000),1000000))
 
+        corrected_bearing = bearing + offset_bearing
+        if corrected_bearing < 0:
```

Comment:
We can actually do `corrected_bearing %= 360` here. It will handle both >360 and <0. Would you like to submit another pull request with the correction?